source/MRMesh/MRContoursCut.cpp

```diff
@@ -2190,6 +2191,36 @@ CutMeshResult cutMesh( Mesh& mesh, const OneMeshContours& contours, const CutMes
     return res;
 }
 
+Expected<FaceBitSet> cutMeshByPolyline( Mesh& mesh, const AffineXf3f& meshXf,
+                                        const Polyline3& polyline, const AffineXf3f& lineXf )
+{
+    const auto xf = meshXf.inverse() * lineXf;
```

Comment:
probably better pass only this `xf` as function argument instead of two: `meshXf` and `lineXf`?